plugins/client/graphql-kotlin-client-generator/src/test/data/generator/shared_response_types/cross_operation_reuse/responses/DetailsObject2.kt

```diff
@@ -0,0 +1,17 @@
+package com.expediagroup.graphql.generated.responses
```

Comment:
place these files in plugins/client/graphql-kotlin-client-generator/src/test/data/generator/cross_operation_reuse_types